Automattic/newspack-plugin
# fix: remove Stripe migration scripts

### All Submissions:






### Changes proposed in this Pull Request:



Moves CLI scripts needed for Stripe migration processes to an external plugin, so those scripts aren't tied to our production release cycle.

### How to test the changes in this Pull Request:

Ensure there are no fatals or warnings and smoke test Stripe donations and data syncs to the ESP to ensure they still work.

### Other information:
